### PR TITLE
[fuzzing] jellyfish fuzzers

### PR DESCRIPTION
**Pull request by lightmark**
## Motivation

Convert proptests to fuzzers

### CODE REVIEW

**Comment by msmouse** [comment]
@mimoo  Is the propfuzz still beta? https://fb.workplace.com/groups/2044217752540140/permalink/2332383197056926

File: testsuite/libra-fuzzer/src/fuzz_targets.rs [hunk 1]
```diff
@@ -73,6 +73,9 @@ static ALL_TARGETS: Lazy<BTreeMap<&'static str, Box<dyn FuzzTargetImpl>>> = Lazy
         // Storage
         // Box::new(storage::StorageSaveBlocks::default()),
         Box::new(storage::StorageSchemaDecode::default()),
+        Box::new(storage::JellyfishGetWithProof::default()),
+        Box::new(storage::JellyfishGetWithProofWithDistinctLastNiblle::default()),
```
Comment (mimoo) [inline testsuite/libra-fuzzer/src/fuzz_targets.rs:77]: it's "nibble" no?

**Review by mimoo — APPROVED** [review]
LGTM! Thanks for working on this : )

**Pull request closed without merging**